tools/flaky-test-reporter/slack_notification.go

```diff
@@ -137,6 +136,21 @@ func createSlackMessageForRepo(rd *RepoData, flakyIssuesMap map[string][]*flakyI
 	return message
 }
 
+// getSlackChannel gets the channel(s) to send messages to for a given job configuration
+func getSlackChannel(rd *RepoData) ([]slackChannel, bool) {
```

Comment:
JobConfig is sufficient as input for this function, I personally would even prefer passing repo name and job name explicitly